app/views/results/_controlled_work_next_steps.html.slim

```diff
@@ -1,18 +1,22 @@
+- end_of_journey_feature_flag = FeatureFlags.enabled?(:end_of_journey, @check.session_data)
+
 = pdf_friendly_p_element(t("results.show.controlled_cw_next_steps.intro_paragraph#{'_partner' if @check.partner}"), @is_pdf)
 = yield
-= pdf_friendly_p_element(t("results.show.controlled_cw_next_steps.second_paragraph"), @is_pdf)
 
-= pdf_friendly_h3(t("results.show.controlled_cw_next_steps.prefilled_header"), @is_pdf)
+- unless end_of_journey_feature_flag && @check.controlled? && @model.decision == "eligible" && @is_pdf
```

Comment:
I think you can simplify this - the `controlled_work_next_steps` partial is only rendered if the check is controlled and the result is eligible. So the only thing you need to check here is if the feature flag is enabled.